src/server/db/schema.ts

```diff
@@ -23,9 +30,17 @@ export const users = createTable(
     password: text('password').notNull(),
     createdAt: timestamp('created_at', { withTimezone: true })
       .default(sql`CURRENT_TIMESTAMP`)
-      .notNull()
+      .notNull(),
   },
   (example) => ({
     emailIndex: index('email_idx').on(example.email),
   }),
 );
+
+export const jobs = createTable('jobs', {
+  id: integer('id').primaryKey().generatedByDefaultAsIdentity(),
+  title: varchar('title', { length: 256 }),
+  company: varchar('company', { length: 256 }),
+  location: varchar('location', { length: 256 }),
+  description: text('description'),
+});
```

Comment:
This is good, but I think we're gonna need some more fields, like the URL of the original posting and fields for when a job was created or updated. Here's some code for that that i already have:

url: varchar('name', { length: 256 }),
    createdAt: timestamp('created_at', { withTimezone: true })
      .default(sql`CURRENT_TIMESTAMP`)
      .notNull(),
    updatedAt: timestamp('updated_at', { withTimezone: true }).$onUpdate(
      () => new Date(),
    ),